ethereum/core/src/main/java/org/hyperledger/besu/ethereum/transaction/TransactionSimulator.java

```diff
@@ -163,25 +157,12 @@ public TransactionSimulator(
         callParams.getGasLimit() >= 0 ? callParams.getGasLimit() : header.getGasLimit();
     final Wei gasPrice = callParams.getGasPrice() != null ? callParams.getGasPrice() : Wei.ZERO;
     final Wei value = callParams.getValue() != null ? callParams.getValue() : Wei.ZERO;
-
-    // If GoQuorum privacy enabled, and value = zero, do simulation with max non-zero bytes.
-    // It is possible to have a data field that has a lower intrinsic value than the PTM hash
-    // so this checks the tx as if we were to place a PTM hash (with all non-zero values).
-    // This means a potential over-estimate of gas, rather than the exact cost to run right now.
-    final Bytes payload =
```

Comment:
This isn't needed now?

Reply:
yeah this PR is reverting my previous one